src/tcpip/tcpip.c

```diff
@@ -759,6 +759,7 @@ static void mg_tcpip_rx(struct mg_tcpip_if *ifp, void *buf, size_t len) {
     rx_ip(ifp, &pkt);
   } else {
     MG_DEBUG(("  Unknown eth type %x", mg_htons(pkt.eth->type)));
+    mg_hexdump(buf, len >= 16 ? 16 : len);
```

Comment:
Maybe we could let VLANs pass, ethtype = 6; we know them, we just don't support them (yet ?)

Reply:
I believe you're correct!